main.go

```diff
@@ -334,6 +330,32 @@ func (c *config) setDerivedFields() error {
 		}
 		c.dynamoPath = path.Join(c.goAbsolutePackagePath, *c.relativeDynamoPath)
 	}
+}
 
-	return nil
+func getModulePackagePath(goPath, outputPath string) string {
```

Comment:
This and `getModulePackageName` didn't change, right? You just moved them?

Reply:
yes